config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "GITHUB_CLIENT_ID": "167ea4b09801db1de03d",
+  "GITHUB_CLIENT_SECRET": "e1208e29f4e03c5f9c6fc08a36375df8344ab01d"
+}
```

Comment:
Did you mean to check this in?

Reply:
Nop! This is the config for testing github connection. I'll remove it. Thanks for catching this!